tests/generic_config_updater/conftest.py

```diff
@@ -2,6 +2,35 @@
 
 from tests.common.utilities import skip_release
 
+@pytest.fixture(autouse=True)
+def ignore_expected_loganalyzer_exceptions(duthost, loganalyzer):
+    """
+       Ignore expected yang validation failure during test execution
+
+       GCU will try several sortings of JsonPatch until the sorting passes yang validation
+
+       Args:
+           loganalyzer: Loganalyzer utility fixture
+    """
+    # When loganalyzer is disabled, the object could be None
+    if loganalyzer:
+         ignoreRegex = [
+             ".*ERR sonic_yang.*",
+             ".*ERR bgp#bgpcfgd.*Can't update the peer. Only 'admin_status' attribute is supported.*", # test_bgpl
+             ".*ERR bgp#bgpcfgd: BGPAllowListMgr::Received BGP ALLOWED 'SET' message with no prefixes specified: {'NULL': 'NULL'}.*", # test_bgp_prefix
+             ".*ERR.*Failed to start dhcp_relay container.*", # test_dhcp_relay
+             ".*ERR GenericConfigUpdater: Service Validator: Service has been reset.*", # test_dhcp_relay test_syslog
+             ".*Same listen range is attached to peer-group.*", # test_bgp_speaker -> real issue
+             ".*ERR swss[0-9]*#orchagent.*removeLag.*", # autorestart/test_container_autorestart.py test_portchannel_interface
+             ".*ERR swss[0-9]*#intfmgrd: :- setIntfVrf:.*", # test_portchannel_interface
+             ".*ERR teamd.*get_dump: Can't get dump for LAG.*", # test_portchannel_interface
+             ".*ERR kernel.*Reset adapter.*", # test_portchannel_interface replace mtu
+             ".*ERR swss[0-9]*#orchagent: :- getPortOperSpeed.*", # test_portchannel_interface replace mtu
+             ".*ERR.*Failed to apply Json change.*", # validator need updater submodule
+             ".*ERR GenericConfigUpdater: Change Applier: service invoked.*", # validator need updater submodule
```

Comment:
Are all these errors only be triggered during GCU testing?
If these errors can be observed in other tests, they need to be added to the common ignore list.

Reply:
These error are only for GCU test.